modules/rust-wasm-test/src/lib.rs

```diff
@@ -117,10 +118,37 @@ pub fn test(ctx: ReducerContext, arg: TestAlias, arg2: TestB, arg3: TestC) -> an
 
     log::info!("Row count filtered by condition: {:?}", other_row_count);
 
+    log::info!("MultiColumn");
+
+    for i in 0i64..1000 {
+        Point::insert(Point {
+            x: i + arg.x as i64,
+            y: i + arg.y as i64,
+        });
+    }
+
+    let multi_row_count = query!(|row: Point| row.x >= 0 && row.x <= 200).count();
+
+    log::info!("Row count filtered by multi-column condition: {:?}", multi_row_count);
+
     log::info!("END");
     Ok(())
 }
 
+#[spacetimedb(reducer)]
+pub fn test_query(ctx: ReducerContext) -> anyhow::Result<()> {
+    test(
```

Comment:
Do you need both to be reducers? If you want to just invoke this one, then maybe `test` itself shouldn't have `spacetimedb(reducer)`. Alternatively, just invoke `test` itself from standalone_integration_test.

Reply:
I don't understand the purpose of having `test` and not using it elsewhere, I think we speculate it just was to confirm it compiles. If that is the case having both makes some sense.

Reply:
Yeah but then you can probably just call `test` directly from your test runner, just pass non-empty `args` maybe?

Reply:
Done

Reply:
You happy with this @RReverser ?

Reply:
Yup, thanks.